colibre/auto_plotter/halo_mass_functions.yml

```diff
@@ -1,5 +1,6 @@
 halo_mass_function:
   type: "massfunction"
+  select_structure_type: 10
```

Comment:
Can you please also include `comment: "Centrals only"` for every plot that now has `select_structure_type: 10` so that we know that the plot shows only central objects

Reply:
I can but that should be obvious for a halo mass function.

Reply:
You are right. However, I mentioned it because the pattern followed in the pipeline is that (almost) every plot with the mask `select_structure_type: 10` has a comment "Centrals only", so we always know exactly what is plotted.